### DISCUSSION TITLE
Bots are deleting Slipsuits when leaving the Dugong to a Station

### DISCUSSION THREAD

**Discussion by Kiwibaum** [FakeFishGames/Barotrauma#13630] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I had a slipsuit on and when I switched to a different one the bot climbed up the ladder from the dugong into the starting station I was still docked at. The bot dropped the suit and I got the message about items being dropped by bots being transported to the submarine later. However the suit was just gone and I have no clue where it went. This was reproducable and it didn't happen for PUCS. PUCS just layed in the space between the station and the dugong at the ladder.

### Reproduction steps

1, Be docked in a Dugong at a station (Not tested on other submarines and only on the starting station)
2. Put on a Slipsuit
3. Switch to a different bot
4. Wait till the bot climbs up the ladder towards the station and drops the suit. 

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.2.12.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by giists** [comment]
Unfortunately I wasn't able to reproduce this. Could you send the save file where this bug occurred?
